lib/analytics/course_csv_builder.rb

```diff
@@ -75,6 +73,14 @@ def headers
     end
   end
 
+  def revisions_rows(row)
+    row << @course.revision_count
+    row << revisions_id
```

Comment:
This is, basically, putting all the revision ids as an array into one cell of the CSV? It looks like this also would add this to the other CSVs besides the revision one?

I'm not sure, but I think the revisions list is a different enough set of data that it might make more sense to create a new, dedicate CSV builder file instead of adapting this one to the new use. This one is designed to output one row per course, whereas I think a revisions CSV would ideally be one row per revision.

Reply:
Are you suggesting to make a new file name course_revisions_csv_builder.rb? and fetch revision data for campaign revision CSV file from this file as we do for articles?

Reply:
Following the pattern for articles is the best option, I think.